CONTRIBUTING.md

```diff
@@ -47,19 +47,35 @@ a developer should pick it up.
 
 ### Structure
 
-This documentation follows the [Diátaxis] documentation framework.
+The various documents are divided by topic,
+for example:
+
+-   Getting started
+-   Process database
+-   Modules
+-   etc.
+
+This documentation also follows the [Diátaxis] documentation framework.
 We recommend reading the [Diátaxis] documentation.
```

Comment:
Very minor (and not from this change), it seems odd to have the same link twice in back-to-back sentences.